2021/scripts.js

```diff
@@ -115,7 +115,7 @@ window.addEventListener("load", async () => {
       lang: _ => "rust",
       solutionUrl: (day, part) => {
         const _day = (day + 1).toString().padStart(2, "0");
-        return `https://raw.githubusercontent.com/Skgland/Advent-of-Code-2021/main/year2021/src/day${_day}.rs`
+        return `https://raw.githubusercontent.com/Skgland/Advent-of-Code-2021/main/crates/year2021/src/day${_day}.rs`
```

Comment:
You forgot to change the year in your repository name.

Reply:
yeah, looks like I never updated it when I removed the year from the year from the repo name

Reply:
fixed